bookserver/crud.py

```diff
@@ -118,6 +118,39 @@ async def count_useinfo_for(
         return res.all()
 
 
+async def fetch_page_activity_counts(
+    chapter: str, subchapter: str, base_course: str, course_name: str, username: str
+) -> Dict[str, int]:
+    query = select(Question).where(
```

Comment:
Add a comment, perhaps "Find all questions on the given page (defined as a specific chapter and subchapter of a book)".